### ISSUE TITLE
Add a CI/CD tool to measure simple language page content

### ISSUE THREAD

**Issue by alisonhall** [PublicisSapient/enable-a11y#104]
Investigate adding a command line tool to measure the language level of a page. The text content of each page should be simple ([WCAG requirement 3.1.5](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-supplements.html)), and having a tool to measure the reading level of each page would be useful.

If a good command line tool can be found, it should be integrated into the GitHub Actions CI/CD pipelines.

Example of tool: https://askubuntu.com/questions/1124127/tool-for-measuring-readability-of-english-text
